### ISSUE TITLE
Sentinel authentication

### ISSUE THREAD

**Issue by simsicon** [aio-libs/aioredis-py#690]
Redis [supports sentinel authentication](https://redis.io/topics/sentinel#configuring-sentinel-instances-with-authentication) since 5.0.1.

When I use `aioredis.create_sentinel` to initialize a sentinel instance, the password arg is for redis backends, not for sentinel password.

redis-py [provides](https://github.com/andymccurdy/redis-py/issues/1256#issuecomment-575038858) `sentinel_kwargs` to pass the arg to sentinel.

Is there a similar way to do that? Or how can I quick fix this? Thanks.

**Comment by mohammadhzp** [comment]
I just ran into the issue and as a quick fix I did this:

```python
from async_timeout import timeout as async_timeout
import asyncio
import aioredis
import aioredis.log


class MySentinelPool(aioredis.sentinel.SentinelPool):
    def __init__(self, *args, **kwargs):
        try:
            self.sentinel_password = kwargs['sentinel_password']
        except KeyError:
            self.sentinel_password = None
        else:
            del kwargs['sentinel_password']

        super().__init__(*args, **kwargs)

    async def _connect_sentinel(self, address, timeout, pools):
        try:
            with async_timeout(timeout):
                pool = await aioredis.pool.create_pool(
                    address, minsize=1, maxsize=2,
                    parser=self._parser_class, password=self.sentinel_password
                )
            pools.append(pool)
            return pool
        except asyncio.TimeoutError as err:
            aioredis.log.sentinel_logger.debug(
                "Failed to connect to Sentinel(%r) within %ss timeout",
                address, timeout)
            return err
        except Exception as err:
            aioredis.log.sentinel_logger.debug(
                "Error connecting to Sentinel(%r): %r", address, err)
            return err
```

**Comment by seandstewart** [comment]
This issue should be resolved as of #891 - please feel free to pull in the latest master and test to ensure this is the case.
